po/fr.po

```diff
@@ -312,40 +312,35 @@ msgstr "Si aucun argument n'est fourni 'paru -Syu' sera exécuté"
 
 #: src/help.rs:26
 msgid "Options without operation:"
-msgstr "Options sans opérations:"
+msgstr "Options sans opérations :"
 
 #: src/help.rs:27
 msgid "    -c --clean            Remove unneeded dependencies"
-msgstr "    -c --clean             Supprime les dépendances inutiles"
+msgstr "    -c --clean            Supprime les dépendances inutiles"
 
 #: src/help.rs:28
-msgid ""
-"       --gendb            Generates development package DB used for updating"
-msgstr ""
-"       --gendb             Génère la BDD de paquets de développement "
-"utilisée pour mettre à jour"
+msgid "       --gendb            Generates development package DB used for updating"
+msgstr "       --gendb            Génère la BDD de paquets de développement utilisée pour mettre à jour"
 
 #: src/help.rs:30
 msgid "New options:"
-msgstr "Nouvelles options:"
+msgstr "Nouvelles options :"
 
 #: src/help.rs:31
 msgid "       --repo              Assume targets are from the repositories"
-msgstr "       --repo              Suppose que les cibles sont dans les depôts"
+msgstr "       --repo             Suppose que les cibles sont dans les depôts"
```

Comment:
Why did you remove this? It pushes everything back and misalignes it.

Reply:
I removed the space so everything (except the flags from --makepkg to --fmflags included) is aligned. The <n> of --completioninterval is aligned with the other flags descriptions

Reply:
That makes there an only 1 space gap between `combinedupgrade` though where there should be at least two. The misalignment there probably happened when the `--[no]`s were added. But anyway just stick to the english layout and that can be fixed another time.

Reply:
I fixed the spaces